index.html

```diff
@@ -226,76 +226,39 @@ <h1>The Parameter Registry</h1>
 
   <section>
     <h1>The Property Registry</h1>
-
     <p>
-This table summarizes the globally registered properties that can be provided
-as properties in a DID Document.
+      The following section defines the properties in a DID document.
     </p>
-
-    <table class="simple">
-      <thead>
-        <tr>
-          <th>Property</th>
-          <th>Description</th>
-        </tr>
-      </thead>
-
-      <tbody>
-        <tr>
-          <td>
-TBD
-          </td>
-          <td>
-            <div>DESCRIPTION_GOES_HERE</div>
-            <div><a href="">SPECIFICATION_GOES_HERE</a></div>
-            <pre>
-"@context": [
-  "https://www.w3.org/ns/did/v1",
-  "https://CONTEXT_GOES_HERE/v1"
-]
-            </pre>
-          </td>
-        </tr>
-      </tbody>
-    </table>
-  </section>
-
-
     <section>
-      <h1>The Value Registry</h1>
```

Comment:
Don't delete The Value Registry... we need a way of specifying right-hand-side values... like "Ed25519VerificationKey2018" and "XyzService".

Reply:
Why? can't we just define what "type" is and provide specific examples of it using JSON Schema and JSON-LD? The usability of a registry with split key value tables is really rough... I worry that if it is not nice looking and usable, there will be no contribution.

Reply:
Additionally, if we generate this document from JSON-Schema and JSON-LD, we can directly leverage those structures to make these links clear in a single location.

Reply:
> Why? can't we just define what "type" is and provide specific examples of it using JSON Schema and JSON-LD?

There are currently 32 possible types that could appear... I guess we can provide some gigantic list of them, but they need descriptions too. How is that going to work? The type section ends up being multiple pages in length? Is that the plan? If you want to try it that way, I'm game... we've been doing it like so for a while: https://w3c-ccg.github.io/security-vocab/#classes

I'm not saying we want to do it that way, but it certainly helps to not dump a whole bunch of stuff into the 'type' property.

Reply:
yes, I see the type section getting very long, seeing how long it gets is helpful reminder that optionality is costly... I think it can be achieved in a readable manner though. i need to provide example definitions for all the parts of a publicKey in this PR, so we can get a sense of what this will be.

Reply:
> I think it can be achieved in a readable manner though.

Ok, I'm fine to try it as long as we're both aware that it might not work out.

Reply:
We should get consensus while the definitions are small, before adding anything else, that way its easy to change if people don't like it.

Reply:
If we need to add the value registry back we can, I'd prefer to handle this with json schema instead of tables though... can we resolve this?